### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by jbaxleyiii**
Moved the (empty) docs into this repo and setup auto deploys with netlify!

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - index
+
+github_repo: apollographql/apollo-android
+content_root: docs/source
+
+social_links:
+  github: 'https://github.com/apollographql'
+  twitter: '@apollographql'
+
+# API keys
```
Comment (digitalbuddha) [inline docs/_config.yml]: Sanity check, not sure what this is exactly but  should api keys be checked into public repo?

File: docs/circle.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+machine:
```
Comment (digitalbuddha) [inline docs/circle.yml]: Currently we use Travis for CI, do we need to introduce circle as well?

Comment (stubailo) [reply docs/circle.yml]: No - this file is unnecessary

**Comment by stubailo** [comment]
I'm going to merge this for now. We're not using this URL yet for anything, but I want to finish up the migration of the repos etc. -- this merge doesn't mean we need to start using this site for the docs or anything.

**Pull request merged**